lib/fs.js

```diff
@@ -72,36 +72,24 @@ const BufferToString = uncurryThis(Buffer.prototype.toString);
 const {
   AbortError,
   aggregateTwoErrors,
-  codes: {
-    ERR_ACCESS_DENIED,
```

Comment:
Can you make sure you don't change any lines that is irrelevant to your PR?

Reply:
Aghhh, sorry this was lsp auto saving